src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -69,97 +235,135 @@ public void runCargo(boolean a,boolean reversed) {
 
   public boolean wheelReady(){
     double flywheelSpeed = KShooterEncoder.getVelocity();
+    currentRPM = DShooterRPMInput.getDouble(0);
```

Comment:
Set currentRPM after/in your aimMode switch statement, since it isn't always this Shuffleboard value in other aim modes.